frontend/src/v5/ui/components/shared/teamspaceNavigation/teamspaceNavigation.component.tsx

```diff
@@ -14,28 +14,21 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Link, useRouteMatch } from 'react-router-dom';
-import { DashboardListItem } from '@components/dashboard/dashboardList';
-import {
-	DashboardListItemRow,
-	DashboardListItemText,
-} from '@components/dashboard/dashboardList/dashboardListItem/components';
-
+import { useRouteMatch } from 'react-router-dom';
 import { discardSlash } from '@/v5/services/routing/routing';
+import { FormattedMessage } from 'react-intl';
+import { Container, Link } from './teamspaceNavigaton.styles';
 
-export const ProjectListItem = ({ projectId, name }): JSX.Element => {
+export const TeamspaceNavigation = (): JSX.Element => {
 	let { url } = useRouteMatch();
 	url = discardSlash(url);
 
 	return (
-		<DashboardListItem>
-			<Link to={`${url}/${projectId}`}>
-				<DashboardListItemRow>
-					<DashboardListItemText>
-						{name}
-					</DashboardListItemText>
-				</DashboardListItemRow>
-			</Link>
-		</DashboardListItem>
+		<Container>
+			<Link to={`${url}/projects`}><FormattedMessage id="teamspaceNavigation.projects" defaultMessage="Projects" /></Link>
+			<Link to={`${url}/settings`}><FormattedMessage id="teamspaceNavigation.settings" defaultMessage="Settings" /></Link>
+			<Link to={`${url}/users`}><FormattedMessage id="teamspaceNavigation.users" defaultMessage="Users" /></Link>
+			<Link to={`${url}/jobs`}><FormattedMessage id="teamspaceNavigation.jobs" defaultMessage="Jobs" /></Link>
```

Comment:
Lets keep the rest of the routing but because in this instance we have only projects defined lets get rid of the rest of the tabs.

Reply:
Done